doc/errorcodes.rst

```diff
@@ -0,0 +1,80 @@
+Other categories currently covered are:
+
+
++------------+----------------------------------------------------------------------------------------------+
+| | Error code                          | Description                                                       |
++------------+----------------------------------------------------------------------------------------------+
+|| ``A001``  |  Abbreviation corrected.                                                                     |
++------------+----------------------------------------------------------------------------------------------+
+|| ``A002``  |  Token corrected as an acronym.                                                              |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C001``  |  Duplicated word removed. Should be corrected.                                               |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C002``  |  Wrongly compounded words split up. Should be corrected.                                     |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C003``  |  Wrongly split compounds united. Should be corrected.                                        |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C004``  |  Duplicated word marked as a possible error. Should be pointed out but not deleted.          |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C005``  |  Possible split compound, depends on meaning/PoS chosen by parser.                           |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C006``  |  A part of a compound word is wrong.                                                         |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C007``  |  A multiword compound such as "skóla-og frístundasvið" correctly split up                    |
++------------+----------------------------------------------------------------------------------------------+
+|| ``E001``  |  Unable to parse sentence, possibly something wrong with it.                                 |
++------------+----------------------------------------------------------------------------------------------+
+|| ``E004``  |  The sentence is probably not in Icelandic.                                                  |
++------------+----------------------------------------------------------------------------------------------+
+|| ``N001``  |  Wrong quotation marks                                                                       |
```

Comment:
Fix consistency in using periods at end of description.

Reply:
Good point, done.